chapters/division_of_work/divisiton_of_work.tex

```diff
@@ -0,0 +1,56 @@
+\chapter{Division of work}\label{ch:division_of_work}
```

Comment:
I found the term "work distribution" used more often (for example https://tryinch.com/blog/work-distribution/) can we use this instead?